.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 
 env:
-  go-version: "1.19.7"
+  go-version: "1.20.2"
```

Comment:
I would prefer not specifying minor versions.

The only reason it exists is because of govulncheck; Go 1.20.3 comes out and it will mark all our builds as failing for about a week until GH updates the default version from Go 1.20.2 to Go 1.20.3

and this way we have to do this manually every 3-4 weeks.